assets/src/application/ui/calendars/dateDisplay/CalendarDateRange/index.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { format, parseISO, isValid } from 'date-fns';
+import { __ } from '@wordpress/i18n';
+
+import { BiggieCalendarDate, MediumCalendarDate, CalendarDateProps } from '../index';
+
+export interface CalendarDateRange extends CalendarDateProps {
+	startDate: Date;
+	endDate: Date;
+}
+
+/**
+ * Displays a pair of calendar dates representing a date range
+ */
+export const CalendarDateRange: React.FC<CalendarDateRange> = ({
+	startDate,
+	endDate,
+	headerText = '',
+	footerText = '',
+	className = '',
+	showTime = false,
+}) => {
+	const startDateObject = startDate instanceof Date ? startDate : parseISO(startDate);
+	const endDateObject = endDate instanceof Date ? endDate : parseISO(endDate);
+	if (!isValid(startDateObject) || !isValid(endDateObject)) {
+		return null;
+	}
+	if (format(startDateObject, 'yy-MM-dd') !== format(endDateObject, 'yy-MM-dd')) {
```

Comment:
can this conditional be extracted into a variable?

Reply:
sure... in fact... i think there's a date helper function that checks if dates match